macros/edr/dbt_artifacts/get_dbt_artifacts_materialization.sql

```diff
@@ -0,0 +1,14 @@
+{% macro get_dbt_artifacts_materialized() %}
+    {{ return(adapter.dispatch('get_dbt_artifacts_materialized', 'elementary')()) }}
+{% endmacro %}
+
+{% macro default__get_dbt_artifacts_materialized() %}
+    {# Incremental - to avoid dropping and creating the table #}
+    {% do return('incremental') %}
```

Comment:
I verified - and the default incremental strategy is append or merge - so it should be safe on concurrent runs